plugins/neat.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: neat
+spec:
+  caveats: |
+    This plugin depends on the following binaries -
+    `jq`, `yq` and `sponge` (from `moreutils`), which should be installed by the user
+  description: |
+    If you try to `kubectl get` resources you have just created,
+    they be unreadably verbose. `kubectl-neat` cleans up that redundant information.
+    Can be used as a `kubectl get` wrapper, or by piping into it.
+    For example `kubectl neat pod mypod -o yaml` or
+    `kubectl get pod mypod -o yaml | kubectl neat`.
+  homepage: https://github.com/itaysk/kubectl-neat
+  platforms:
+  - bin: ./kubectl-neat
+    files:
+    - from: /dist/darwin/*
+      to: .
+    selector:
+      matchLabels:
+        arch: amd64
+        os: darwin
+    sha256: 84ddfc9de3301b0359566d5d909aa2c8302aefd48cc8dd107f720be3e3edc24d
+    uri: https://github.com/itaysk/kubectl-neat/releases/download/v0.1.1/kubectl-neat_darwin.tar.gz
+  - bin: ./kubectl-neat
+    files:
+    - from: /dist/linux/*
+      to: .
+    selector:
+      matchLabels:
+        arch: amd64
+        os: linux
+    sha256: 18066653dc535646f940e8ed09f1d68eccdd63e6640eba5d8160544eec6a4f4f
+    uri: https://github.com/itaysk/kubectl-neat/releases/download/v0.1.1/kubectl-neat_linux.tar.gz
+  shortDescription: Clean up Kubernetes yaml and json output to make it readable.
```

Comment:
This is just a suggestion, maybe you like the wording:

> Remove clutter from manifests

or

> Remove clutter from manifests for readability

Reply:
Thanks, updated